app/code/Magento/Store/Api/StoreConfigManagerInterface.php

```diff
@@ -17,8 +17,8 @@
 interface StoreConfigManagerInterface
 {
     /**
-     * @param string[] $storeCodes
+     * @param string $storeCodes
      * @return \Magento\Store\Api\Data\StoreConfigInterface[]
      */
-    public function getStoreConfigs(array $storeCodes = null);
+    public function getStoreConfigs($storeCodes = null);
```

Comment:
Hi @lakshaypanwar. Changing signatures of public methods of @api interfaces is prohibited. There might be 3rd party implementations of the current interfaces that implement the public class with the current signature. If we change the interface in this way, we will an error to an existing Magento installation. 
You can get more details about backward compatible development from the [following](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/) article.